packages/gatsby-source-wordpress/src/steps/create-schema-customization/helpers.js

```diff
@@ -20,40 +64,66 @@ export const buildTypeName = name => {
     return name
   }
 
+  // Gatsby makes the same type, so we need to rename it to prevent conflicts
   if (name === `Filter`) {
     name = `FilterType`
   }
 
+  if (
+    // Starting in WPGraphQL 1.13.0, Gatsby and WPGraphQL both generate types ending in these strings for every node type in the schema, so we need to rename types to prevent conflicts.
+    // for users on older versions of WPGraphQL we should try to keep the schema how it was before
+    isWpgqlOneThirteenZeroPlus() &&
+    (name.endsWith(`Connection`) ||
+      name.endsWith(`Edge`) ||
+      name.endsWith(`PageInfo`))
+  ) {
+    name += `Type`
+  }
+
```

Comment:
This was one of the major problems, Gatsby and source-wordpress (on latest WPGraphQL) were both making types like `Wp${type}Connection` and `Wp${type}Edge`. The wpgql types were interfaces while the Gatsby ones are object types and this caused graphql-js to throw errors about multiple types with the same name.